itests/docker/Dockerfile

```diff
@@ -1,11 +1,12 @@
 ARG MARATHONVERSION=v1.6.322
```

Comment:
do we still need this?

Reply:
Yes, we still need it because of this https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact